2 table.sql

```diff
@@ -0,0 +1,27 @@
+use ArmyDB
+go
+drop table if exists Soldiers
+go
+create table dbo.Soldiers(
+	SoldierID int not null identity primary key,
+	SoldierFirstName varchar(30) not null
+		constraint c_Soldiers_Soldier_First_Name_cannot_be_blank check (SoldierFirstName <> ''),
+	SoldierLastName varchar(30) not null
+		constraint c_Soldiers_Soldier_Last_Name_cannot_be_blank check (SoldierLastName <> ''),
+	SSN char(10) not null
```

Comment:
There's a much easier way to do this constraint. Use like.

Reply:
Changed to like

Reply:
Use `SSN like [1-9][0-9]`...

Reply:
Use `SSN like [1-9][0-9]`...